apps/Gruntfile.js

```diff
@@ -363,6 +363,8 @@ module.exports = function (grunt) {
   config.exec = {
     convertScssVars: './script/convert-scss-variables.js',
     generateSharedConstants: 'bundle exec ./script/generateSharedConstants.rb',
+    generateRegionConfigurations:
+      'bundle exec ./script/generateRegionConfigurations.rb',
```

Comment:
Converts the `config/global_editions/*.yml` into `json` and makes it available in JS-land.